packages/prometheus_pint/package.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: prometheus_pint
+description: Prometheus rule linter/validator
+homepage: https://cloudflare.github.io/pint/
+licenses:
+  - Apache-2.0
+languages:
+  - PromQL
+categories:
+  - Linter
+
+source:
+  id: pkg:github/cloudflare/pint@v0.73.7
+  asset:
+    - target: linux_x86
+      file: pint-{{ version | strip_prefix "v" }}-linux-386.tar.gz
+      bin: pint-linux-386
+    - target: linux_x64
+      file: pint-{{ version | strip_prefix "v" }}-linux-amd64.tar.gz
+      bin: pint-linux-amd64
+    - target: linux_arm64
+      file: pint-{{ version | strip_prefix "v" }}-linux-arm64.tar.gz
+      bin: pint-linux-arm64
+    - target: darwin_x64
+      file: pint-{{ version | strip_prefix "v" }}-darwin-amd64.tar.gz
+      bin: pint-darwin-amd64
+    - target: darwin_arm64
+      file: pint-{{ version | strip_prefix "v" }}-darwin-arm64.tar.gz
+      bin: pint-darwin-arm64
+    - target: win_x86
+      file: pint-{{ version | strip_prefix "v" }}-windows-386.tar.gz
+      bin: pint-windows-386.exe
+    - target: win_x64
+      file: pint-{{ version | strip_prefix "v" }}-windows-amd64.tar.gz
+      bin: pint-windows-amd64.exe
+
+bin:
+  prometheus-pint: "{{source.asset.bin}}"
```

Comment:
Sorry for nitpicking but shouldn't it be an underscore to match the package name?

Reply:
And that breaks some linting for some reason.

Reply:
I don't know what to do about it.